client/js/libs/handlebars/parse.js

```diff
@@ -83,6 +91,9 @@ module.exports = function parse(text) {
 			return `<span class="inline-channel" role="button" tabindex="0" data-chan="${escapedChannel}">${fragments}</span>`;
 		} else if (textPart.emoji) {
 			return `<span class="emoji">${fragments}</span>`;
+		} else if (textPart.nick) {
+			const nick = fragments.trim();
+			return `<span role="button" class="user ${colorClass(nick)}" data-name="${nick}">${nick}</span>`;
```

Comment:
`fragments` may contain HTML, so you probably want to use `textPart.nick` in `colorClass`/`date-name` (and escape it?)

Reply:
Don't think I need to escape?